src/v/schema/utils.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2024 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+
+#include "pandaproxy/schema_registry/types.h"
+#include <optional>
+namespace schema_registry {
```

Comment:
this appears to be unused in this commit, and orthogonal to the scope of this commit of "move wasm::schema_registry to schema_registry".

Reply:
Also wrong namespace